src/commands/BootstrapCommand.js

```diff
@@ -49,24 +50,24 @@ export default class BootstrapCommand extends Command {
       const linkSrc = dependency.location;
       const linkDest = path.join(pkg.nodeModulesLocation, dependency.name);
 
-      this.createLinkedDependency(linkSrc, linkDest, dependency.name, done);
+      this.createLinkedDependency(linkSrc, linkDest, dependency.name, (err) => {
+        if (!err && dependency._package.bin) {
```

Comment:
Great!  So it should be easy to back port. 😁 

The functionality provided by _this_ PR is independently useful and, frankly, less controversial than #237.  I don't want it to get delayed by association.  I'd like to see this merge, independently, as soon as possible.  That would have the added benefit of reducing the scope of #237, which is pretty big.